sling/nlp/parser/tools/commons_from_corpora.py

```diff
@@ -74,15 +74,19 @@ def add(handle):
                default="",
                type=str,
                metavar='COMMA_SEPARATED_RECORDIOS')
-
   flags.define('--output',
                help='Output commons file name',
                default="/tmp/commons.sling",
                type=str,
                metavar='OUTPUT_COMMONS_FILENAME')
+  flags.define('--binary',
```

Comment:
I think we should have binary=True as the default.

Reply:
Done.